docs/STREAMING.md

```diff
@@ -0,0 +1,75 @@
+# Streaming 
+
+
+Uses grpc streams to allow the plugin to send data immediately or after a certain period of time instead of on an interval governed by Snap.
+Streaming by snap enables - 
+* Improved performance by enabling event based data flows
+* Supports runtime configuration controlling event throughput
```

Comment:
Can you get rid of `Supports` and start the sentence with `Runtime configuration to control event throughput`